### ISSUE TITLE
Secret the API credentials with the build process

### ISSUE THREAD

**Issue by hoangnguyen92dn** [miles-pudge-halter/nimble-survey-android#2]
- [ ]  Secret the API credentials with the build process

With your current implementation, the API credentials could be leaked by reverse engineering. I'd suggest storing the credentials in the configuration file rather than in the project codebase; with that config, we could secret the credentials with the build process; how do you think?

**Comment by miles-pudge-halter** [comment]
We can have it in a gitignored file like `local.properties` and then add it as a BuildConfig field.

**Comment by hoangnguyen92dn** [comment]
@miles-pudge-halter Yup. That could be a solution to resolve the issue 👍  And please don't hesitate to open the PR to apply your fixes on submitted issues, feel free to ask if you have any concern.
